modules/apps/analytics/analytics-settings-rest-impl/src/main/java/com/liferay/analytics/settings/rest/internal/resource/v1_0/FieldResourceImpl.java

```diff
@@ -477,6 +468,30 @@ private List<Field> _sort(List<Field> fields, Sort[] sorts) {
 		return fields;
 	}
 
+	private Field _toField(
+		String[] examples, int index, String[] names, String[] requiredNames,
```

Comment:
This is weird. Why don't you just pass the String example, the String name, etc without accessing to the index inside the method?

Reply:
I think it is even simpler to get rid of this method and just creating the field inside the for loop